gluster-exporter/metric_brick.go

```diff
@@ -567,7 +567,7 @@ func lvmUsage(path string) (stats []LVMStat, thinPoolStats []ThinPoolStat, err e
 				continue
 			}
 			// Check if the logical volume is mounted as a gluster brick
-			if lv.Device == dev && strings.HasPrefix(path, mount.Name) {
+			if lv.Device == dev && strings.HasPrefix(path, mount.Name) && strings.Contains(path, lv.VGName) && strings.Contains(path, lv.Name) {
```

Comment:
hyphens in VGName and LVName are converted to double hyphen in `/proc/mounts`. That needs to be handled.

Reply:
I print the /proc/mounts lv.VGName, but i can not found double hyphen
I use the "/" connection to be more accurate @aravindavk 
https://github.com/gluster/gluster-prometheus/blob/fbebfc309146f632834f3f6be05fd2e1422f8c26/gluster-exporter/metric_brick.go#L570

**print code**
```
				fmt.Println("VGName+lvName:","/"+lv.VGName+"/"+lv.Name)
				fmt.Println("path:",path)
				fmt.Println("mount.Name:",mount.Name)
				fmt.Println()
```
**the print result:**
![image](https://user-images.githubusercontent.com/20083081/58230604-615b7d00-7d67-11e9-9581-502b77f017c8.png)